### ISSUE TITLE
Issue in parsing value ranges 

### ISSUE THREAD

**Issue by nickaein** [cederberg/mibble#33]
There seems to be an issue in parsing value-range constraints. Consider the following toy MIB:

```
MY-MIB DEFINITIONS ::= BEGIN

IMPORTS OBJECT-TYPE, Integer32 FROM SNMPv2-SMI;

my-parameter OBJECT-TYPE
	SYNTAX 	Integer32 (100..200)
	MAX-ACCESS  	read-only
	STATUS 	current
	DESCRIPTION "Simple parameter"
	::= { iso 1 }

END
```

The [exitValueConstraint](https://github.com/cederberg/mibble/blob/master/src/java/net/percederberg/mibble/MibAnalyzer.java#L1155) method process the constraints and eventually creates an `ValueRangeConstraint` instance with lowerbound of 200 and no upperbound. There is possibly an issue inside `for` loop in determining the presence of low and upper bounds.

**Comment by cederberg** [comment]
Good catch. Issue is on [line 1179](https://github.com/cederberg/mibble/blob/47314e8f3a0c2d99e6092ab07ceb9067c0d6ea5f/src/java/net/percederberg/mibble/MibAnalyzer.java#L1179) where `strictLower` should be `lower`. Must have been tumbled around in one refactoring or another.

**Comment by nickaein** [comment]
That seems to take care of it. Thanks for quick update!
